build/stage-build.yml

```diff
@@ -75,44 +70,102 @@ jobs:
     pool:
       name: $(windowsPoolName)
 
+    steps:
+      - template: steps-build-windows.yml
+        parameters:
+          pathToSrc: ${{ parameters.pathToSrc }}
+          solutionFileName: ${{ parameters.solutionFileName }}
+
+
+  - job: OnWindows_Android
+    strategy:
+      matrix:
+        Android:
+          ApplicationPlatform: AnyCPU
+          ArtifactName: Android
+          BuildForPlayStore: False
+          VersionCodeOffset: 1
+
+    pool:
+      name: $(windowsPoolName)
+    
     variables:
     - group: ${{ parameters.androidVariableGroup }}
 
     container: windows
 
     steps:
-      - template: steps-build.yml
-        parameters:
-          pathToSrc: ${{ parameters.pathToSrc }}
-          solutionFileName: ${{ parameters.solutionFileName }}
-          androidKeyStoreFile: ${{ parameters.androidKeyStoreFile }}
-          pathToAndroidManifest: ${{ parameters.pathToAndroidManifest }}
+    - task: DownloadSecureFile@1
+      name: keyStore
+      displayName: "Download keystore from secure files"
+      inputs:
+        secureFile: ${{ parameters.androidKeyStoreFile }}
+
+    #- task: android-manifest-package-name@1
+      #displayName: "Update bundle identifier"
+      #inputs:
+        #sourcePath: ${{ parameters.pathToAndroidManifest }}
+        #packageName: $(ApplicationIdentifier)
+        #appLabel: $(AppLabel) # The label of the app (displayed in launcher and in the OS)
+        #printFile: true
 
-  - job: On_Mac # Build iOS on a mac
+    - task: android-manifest-version@1
+      displayName: "Bump android manifest version"
+      inputs:
+        sourcePath: ${{ parameters.pathToAndroidManifest }}
+        versionCode: $(PreReleaseNumber)
+        versionName: $(MajorMinorPatch)
+        printFile: true
+
+    - template: steps-build-android.yml
+
+  - job: OnMac_iOS
     strategy:
-      maxParallel: 3
+      maxParallel: 2
```

Comment:
@Soap-141 Since there's only one, we can remove the maxParallel right?

Reply:
Didn't event know what it was. I can try.

Reply:
It's great for the win machine when we build android, uwp/windows and test at the same time :)

Reply:
@Soap-141  You can remove maxParallel for mac since it's not needed :)